crates/libs/windows/src/core/heap.rs

```diff
@@ -14,6 +14,13 @@ pub fn heap_alloc(bytes: usize) -> Result<RawPtr> {
     if ptr.is_null() {
         Err(E_OUTOFMEMORY.into())
     } else {
+        // HeapAlloc is not guaranteed to return zero memory but usually does. This just ensures that
+        // it predictably returns non-zero memory for testing purposes. This is similar to what MSVC's
+        // debug allocator does for the same reason.
+        #[cfg(debug_assertions)]
+        unsafe {
+            std::ptr::write_bytes(ptr, b'?', bytes);
```

Comment:
Think it'd be more useful to devs if this returned `asm { int 3 }`
```suggestion
            std::ptr::write_bytes(ptr, 0xCC, bytes);
```